### DISCUSSION TITLE
Unable to find any supported Python versions

### DISCUSSION THREAD

**Discussion by DangSauQuaKhu** [vercel/community#6283] [Help]
### Summary

Looking forward to having your help. I'm having trouble deploying a python app, I'm using version 3.9.18. Even though I see version 3.9 is supported, I still get this error: "Unable to find any supported Python versions.". What can I do?

### Steps to Reproduce

![error](https://github.com/vercel/community/assets/97039865/f3eef911-c445-41e0-ba44-6261fc147491)

**Comment by amyegan** [comment]
Hey @DangSauQuaKhu. Thanks for reporting this! I have a workaround for you to try. Go into project settings and change the Node.js version to 18.x, then redeploy or start a new deployment. Does that fix it?
